src/domain/community/virtualContributor/vcSettingsPage/VCSettingsPage.tsx

```diff
@@ -13,11 +17,31 @@ export const VCSettingsPage = () => {
     },
   });
 
+  const [updateContributorMutation] = useUpdateVirtualContributorMutation();
+
+  const handleUpdate = async virtualContributor => {
+    await updateContributorMutation({
+      variables: {
+        virtualContributorData: {
+          ID: virtualContributor.ID,
+          profileData: virtualContributor.profileData,
+        },
+      },
+    });
+  };
+
   if (loading) return <Loading text={'Loading Virtual Contributor Settings ...'} />;
 
+  // TODO: StorageProvider for the VC
```

Comment:
Normally we just use the platform StorageProvider for entities that we are creating (which is definitely wrong, but it is a solution if you need it now), 
or hide the ReferencesSegment in the Create form and show it in the edit form. Which is also bad but I think it's better better.